crates/astria-bridge-withdrawer/src/withdrawer/ethereum/test_utils.rs

```diff
@@ -0,0 +1,68 @@
+use std::{
+    path::Path,
+    sync::Arc,
+    time::Duration,
+};
+
+use ethers::{
+    core::utils::Anvil,
+    prelude::*,
+    utils::AnvilInstance,
+};
+
+/// Starts a local anvil instance and deploys the `AstriaWithdrawer` contract to it.
+///
+/// Returns the contract address, provider, wallet, and anvil instance.
+///
+/// # Panics
+///
+/// - if the contract cannot be found in the expected path
+/// - if the contract cannot be compiled
+/// - if the provider fails to connect to the anvil instance
+/// - if the contract fails to deploy
+#[allow(dead_code)]
+pub(crate) async fn deploy_astria_withdrawer()
```

Comment:
Delete. We don't want dead code.

Reply:
this isn't dead code, will remove the allow